src/modules/menu/components/search/SearchResultList.vue

```diff
@@ -38,11 +36,17 @@ function gotToLayerCategory() {
     layerCategory.value.focusFirstEntry()
 }
 
+function gotToLayerFeaturesCategory() {
+    layerFeatureCategory.value.focusFirstEntry()
+}
+
 function focusFirstEntry() {
-    if (results.value.locationResults?.length) {
-        locationCategory.value.focusFirstEntry()
-    } else if (results.value.layerResults?.length) {
-        layerCategory.value.focusFirstEntry()
+    if (locationResults.value.length) {
+        gotToLocationCategory()
+    } else if (layerResults.value.length) {
+        gotToLayerCategory()
+    } else if (layerFeatureResults.value.length) {
+        gotToLayerFeaturesCategory()
     }
```

Comment:
This logic broke the keyboard navigation, I think we should rename the function above `gotToLocationCategory` to `gotToLocationCategoryLastEntry`